pkg/helm/actions/kube_version.go

```diff
@@ -6,7 +6,7 @@ import (
 	"k8s.io/client-go/rest"
 )
 
-func GetKubeVersion(config *rest.Config) (string, error) {
+func kubeVersion(config *rest.Config) (string, error) {
```

Comment:
this function is not needed anymore, because it is moved under `version` package. Actually, the whole `kube_version.go` file can be removed.

Reply:
done